### ISSUE TITLE
Retrieve Podcast episode links from Spotify, Apple Music, Amazon Music and Google automatically

### ISSUE THREAD

**Issue by andygrunwald** [EngineeringKiosk/webpage#26]
The python script in https://github.com/EngineeringKiosk/webpage/blob/main/scripts/podcast-feed-to-content.pycreates the initial podcast episode entry as Markdown based on the Podcast RSS feed.

The links for the particular platforms (Spotify, etc.) still need to be added manually. See https://github.com/EngineeringKiosk/webpage/blob/bb46c27db352a0d5772a7b72563f4e25b4e63071/scripts/podcast-feed-to-content.py#L270-L273

### Goal of this task

See if we can automate the retrieval of the links from Spotify, Apple Music, Amazon Music and Google for each podcast episode.

**Comment by andygrunwald** [comment]
Spotify done in
- https://github.com/EngineeringKiosk/webpage/commit/a55916cd0bf6f4c40685c7e7926a8e6a9b9a0205
- https://github.com/EngineeringKiosk/webpage/commit/9fb33ed9eb8f914e4fa0cd3c2317c307c504d587

Apple Podcast done in
- https://github.com/EngineeringKiosk/webpage/commit/0bd374bcad76d279e1e3e4d6efd76ab6c1fe203b

**Comment by andygrunwald** [comment]
Google Podcast done in
- https://github.com/EngineeringKiosk/webpage/commit/7343b1f058d89fc4d87bc95b58b6e0be7c5f67f6

**Comment by andygrunwald** [comment]
Amazon Music is a bit more tricky, because it loads the content async and the API is not very accessible from our script (without building a proper session, etc).

I am not sure if it is really worth the effort and putting hours into it to "reverse engineer" it. For now, I did a small automation/workaround:

1. I skip crawling Amazon Music website (due to JS load scripts)
2. I added a small python script to check for all missing player URLs in all podcast episodes in https://github.com/EngineeringKiosk/webpage/commit/aa54487c4f57d716a530f54f7cd53cbbd5807c78
3. I added a new GitHub action that checks once a month which player URLs are missing - If we have missing ones, it fails and we get an email - See https://github.com/EngineeringKiosk/webpage/runs/6954562212?check_suite_focus=true and https://github.com/EngineeringKiosk/webpage/commit/abdf135e1c53ecfc6b84f9a11680e9687170068c
